practicesharp/Calculator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+
+    
+
+
+    class Calculator
+    {
+        public double Calculate(string equation)
+        {
+            Number number= new Number(ref equation);
+            if (equation == "")
+                return number.decimal_value;
+            char op = equation[0];
+            equation=equation.Remove(0,1);
+            return Operator_Decoder.Decode(op).Activate(number.decimal_value, Calculate(equation));
```

Comment:
You chose to use recursion, are you sure it's really required in this situation?
If you can use just a loop, better use a loop. It's easier to understand and a bit less complex in such case as this.
Do you imagine the implication of using recursion on memory in your program instead of a loop?
If yes, good, if no, also good but let's talk about it.